docker/prometheus.yml

```diff
@@ -21,7 +21,7 @@ scrape_configs:
         - "/etc/prometheus/promgen.json"
     relabel_configs:
     - source_labels: [__shard]
-      regex: docker-demo
+      regex: .*
```

Comment:
Instead of doing a regex .*, could actually drop the entire section to have Prometheus read in all the rules. I think it would be best to leave it there commented out as an example so maybe I'll give this a shot :)